.gitignore

```diff
@@ -4,6 +4,7 @@ main
 config/config.yml
 config/*config.yml
 config/config.yml_*
+!config.yml_example_pocket-id
```

Comment:
This is an odd ignore line. There are examples in the repo currently committed with the ignore patten `config/config.yml_*` which seems to have required you to add this negate ignore line in order to continue the file name pattern.

So the question I would ask is: why is the `config/config.yml_*` line present in the .gitignore in the first place.

Could you commit this change without this added line?

Reply:
Indeed. I did the minimal changes but I agree to simplify gitignore.
Here’s an update.